sdk/eventhub/azure-eventhub/azure/eventhub/_pyamqp/aio/_client_async.py

```diff
@@ -850,13 +864,18 @@ async def receive_messages_iter_async(self, timeout=None, on_message_received=No
         :param on_message_received: A callback to process messages as they arrive from the
          service. It takes a single argument, a ~pyamqp.message.Message object.
         :type on_message_received: callable[~pyamqp.message.Message]
+        :param int timeout: Timeout in seconds for which to wait to receive any messages.
```

Comment:
in general for timeouts that are in seconds, we want float :)

```suggestion
        :param float timeout: Timeout in seconds for which to wait to receive any messages.
```